service-providers/itering/25Q1-doc-edu-app-coordinator.md

```diff
@@ -0,0 +1,47 @@
+---
+applicant: Itering
+type: Docs & Education & App Coordinator
+timeline: 2025-Q1
+costs: 12,000 USDC
+payment: 0x0678E7cB36D3b9A591dF7E42CA23074aF977C52B (Arbitrum One)
+---
+
+## Team
+
+### Member
+
+- [Bear Wang](https://github.com/boundless-forest)
```

Comment:
Add Ran, Denny and other support from Itering.

Reply:
Done